egs/wsj/s5/utils/data/segment_data.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2013  Johns Hopkins University (author: Hossein Hadian)
+# Apache 2.0
+
+
+. utils/parse_options.sh
+. ./path.sh
+
+if [ $# != 2 ]; then
+  echo "Usage: "
```

Comment:
This needs a clear usage message. 
Also, the name of this file is confusing since it is not segmenting data. @danpovey might have a better name.

Reply:
Actually, I've been meaning to ask for name suggestions. Maybe `extract_segments_data_dir.sh`?
The other script is text_to_phones.py (which converts word-level transcriptions to phone-level transcriptions). Maybe `word_transcription_to_phone_transcription.py` ?